src/composables/useUserSkin.ts

```diff
@@ -17,9 +17,11 @@ const _toggleSkin = skin => {
   if (skin === LIGHT_MODE) {
     lsSet('skin', DARK_MODE);
     userSkin.value = DARK_MODE;
+    document.documentElement.setAttribute('data-color-scheme', 'dark');
```

Comment:
You can remove this and line 24 and add the following code inside the `useUserSkin()` below `toggleSkin()` 

```
  watchEffect(() => {
    if (userSkin.value === LIGHT_MODE)
      document.documentElement.setAttribute('data-color-scheme', 'light');
    else document.documentElement.setAttribute('data-color-scheme', 'dark');
  });

```

Reply:
Hi,

it might be also possible to use a simple `watch()`:

```
watch(
	() => userSkin.value,
	(skin) => {
		document.documentElement.setAttribute(
			"data-color-scheme",
			skin === LIGHT_MODE ? "light" : "dark",
		);
	},
);
```

this improves readability and might be more consistent than a watchEffect

---
Please excuse me if that's not the right way to contribute

Reply:
@frabrg great, thanks for pointing this out and for the resources!

Reply:
@frabrg the issue is with just using a watcher here is that it won't trigger on mounted, so an addition `onMounted` would be needed. I'm not sure I see why using `watchEffect` is an issue here.

Reply:
@samuveth thanks for your feedback!
Ok I understand and agree with your last comment 

However it seems to me that `watch` can be triggered on mounted if you specify the option:

```
watch(
	() => userSkin.value,
	(skin) => {
		document.documentElement.setAttribute(
			"data-color-scheme",
			skin === LIGHT_MODE ? "light" : "dark",
		);
	},
       { immediate: true }, // <-- here :)
);
```

According to the [doc](https://v3.vuejs.org/api/computed-watch-api.html#shared-behavior-with-watcheffect):

![image](https://user-images.githubusercontent.com/31159436/146828077-3c0328ef-8844-4fc5-8fc3-efe200d71480.png)

Reply:
Ok interesting!